### DISCUSSION TITLE
Why do my colors look a lot more dull than the one on the screenshots?

### DISCUSSION THREAD

**Discussion by on3iro** [EdenEast/nightfox.nvim#424] [Q&A] [answered]
Hey folks, 

I am using neovim 0.9.5 on OSX inside alacritty, my **$TERM** is set  to `alacritty` and truecolors in neovim are active.

However my background seems to be a lot darker than that from the nightfox screenshot and my mode indicator (from lualine) as well.

Here's how it looks on my setup:

![image](https://github.com/EdenEast/nightfox.nvim/assets/8681413/32eef6e5-ad18-45a5-b21b-45b2b698ea5c)

Compared to the one from the screenshot inside the README:

![image](https://github.com/EdenEast/nightfox.nvim/assets/8681413/f9428771-02cd-4a3e-8933-0c7f9200cfd3)

Any idea what could cause this?

**Comment by EdenEast** [comment]
There are sight differences between terminals. The terminal used in the readme is **Wezterm**. Here is an example of me running 4 terminals at the same time with the same neovim setup and getting slightly different results. (Ignore the different fonts did not want to configure all the terminals to use the same font)

![Screenshot 2024-03-18 at 18 03 17](https://github.com/EdenEast/nightfox.nvim/assets/2746374/8c65423b-0a90-47d3-8e59-0feb5a0be16d)

The terminals are as followed:

| Ghostty | Wezterm |
|---|---|
|Alacritty|Kitty|

**Reply by on3iro** [reply]
thanks for the response. That seems to explain the background issue. However I still wonder why the **insert-mode** color seems to be green for me with default settings, while the nightfox screenshot seems to be more akin to turqoise? No matter which terminal I try, I always get a green.

**Reply by EdenEast** [reply, marked as answer]
The screenshot in the readme is in `normal` mode. The screenshot that you provided is in `insert` mode (which is green). Also in the screenshot it was using my custom feline statusline. That is found [here](https://github.com/EdenEast/nightfox.nvim/blob/main/misc/feline.lua).

**Reply by on3iro** [reply]
oh, yeah you are right. Thanks about the feline hint, though, that clears it up :)
